lib/functions/compilation/patch/drivers_network.sh

```diff
@@ -420,7 +419,7 @@ driver_rtl88x2cs() {
 
 	# Wireless drivers for Realtek 88x2cs chipsets
 
-	if linux-version compare "${version}" ge 5.9 && [ "$EXTRAWIFI" == no ]; then
+	if linux-version compare "${version}" ge 5.9 && linux-version compare "${version}" lt 6.1 ; then
```

Comment:
Part of this change seems unrelated to this PR.  To maintain a clean git history, this change probably should be in a separate PR.

Reply:
The EXTRAWIFI=no there was added during #5265 to avoid conflict with rtw88 patches thats applied in 6.1+, but as a consequence of the change, the rtl88x2cs patches were never applied to any kernel from that point onwards. This fixes the same.

Reply:
As the logic there to disable the driver was using EXTRAWIFI, it needed to be replaced by proper condition so that the conflict won't occur. Keeping EXTRAWIFI=no there, would change the meaning of the flag, which is that no network patch will be applied

Reply:
OK, but then your commit message should have described the necessity for this change as well.  "Make it possible to toggle EXTRAWIFI as a way to disable all extra network patches" makes no mention that you are also fixing the rtw88 patches to correctly get applied in 6.1+.  My point is that the commit message should have included this information or the two changes should have been done under two PRs.

Reply:
Makes sense. Will update the commit message

Reply:
Updated the commit message. Hence marking the conversation as resolved. Feel free to open it if the message is still not clear enough